.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -6,19 +6,29 @@ jobs:
 - job: linux
   pool:
     vmImage: ubuntu-16.04
-  timeoutInMinutes: 360
   strategy:
-    maxParallel: 8
     matrix:
-      linux_python3.6:
-        CONFIG: linux_python3.6
-        UPLOAD_PACKAGES: True
+      linux_64_python3.6.____cpython:
+        CONFIG: linux_64_python3.6.____cpython
+        UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: condaforge/linux-anvil-comp7
-      linux_python3.7:
-        CONFIG: linux_python3.7
-        UPLOAD_PACKAGES: True
+      linux_64_python3.7.____cpython:
+        CONFIG: linux_64_python3.7.____cpython
```

Comment:
This `*___cpython` caused a few problems for some combination of platforms/python versions. I now specify python versions explicitly in `conda_build_config.yaml` file. Probably worth to rerender here after the change is done.

Reply:
There currently is no python version specified in the `conda_build_config.yaml` at the moment. Do you want me to manually make the change and rerender?

Do we need to change this back to the original two lines?

Reply:
Yes please. You can use the file from the `event-model` feedstock, but please add more python versions to the list as this package is arch-dependent.
https://github.com/nsls-ii-forge/event-model-feedstock/blob/5d88d0dce9464dcd9c7a91c54eecdd390638d943/recipe/conda_build_config.yaml#L10

Reply:
Manual rerendering is needed then.

Reply:
I can change the rerender command to use the `conda_build_config.yaml` if it exists. Should be a simple change.

Reply:
It should not be an exclusive config. Regular rerender command should be OK. I think we just need to manually update the file, rerender and rerun the bot.

By the way, does the bot keep track of the already opened PRs with the same update?